erpnext/selling/doctype/quotation/quotation.py

```diff
@@ -185,6 +185,13 @@ def update_item(obj, target, source_parent):
 
 	return doclist
 
+def set_expired_status():
+	from datetime import date
```

Comment:
@0Pranav Please remove the import and other two lines which are not required anymore.